README.md

```diff
@@ -195,8 +195,8 @@
 - [Docker](https://www.docker.com/) - Powerful, performs operating-system-level virtualization.
 - [QEMU](https://github.com/qemu/qemu) - Generic and open source machine emulator and virtualizer. 👏
 - [Vagrant](https://github.com/hashicorp/vagrant) - Tool for building and distributing development environments. 👏
-- [VirtualBox](https://www.virtualbox.org/) - Powerful, brimming with terrific features and, best of all, it’s free.
-- [VMWare Fusion](https://www.vmware.com/) - Powerful, commercial VM developed by VMware.
+- [VirtualBox](https://www.virtualbox.org/) - Free, powerful and brimming with terrific features.
```

Comment:
The description here is totally useless and says nothing about what the thing is.